deploy/blobfuse-proxy/blobfuse-proxy.yaml

```diff
@@ -44,7 +44,7 @@ spec:
             apt-get update
             apt-get install -y blobfuse=1.3.6
             # download blobfuse-proxy .deb package
-            wget https://github.com/kubernetes-sigs/blob-csi-driver/raw/master/deploy/blobfuse-proxy/v0.1.0/blobfuse-proxy-v0.1.0.deb -O /tmp/blobfuse-proxy-v0.1.0.deb
+            wget https://github.com/boddumanohar/blob-csi-driver/raw/issues/392/deploy/blobfuse-proxy/v0.1.0/blobfuse-proxy-v0.1.0.deb -O /tmp/blobfuse-proxy-v0.1.0.deb
```

Comment:
instead of creating a separate PR to to change: `boddumanohar` --> `kubernetes-sigs` . Can I update it right now? So that when the PR is merged, it will point to the correct file  itself.

Reply:
ok